src/AccessibilityInsights.Extensions.AzureDevOps/Enums/AzureDevOpsField.cs

```diff
@@ -31,11 +30,11 @@ public static string ToApiString(this AzureDevOpsField value)
             switch (value)
             {
                 // These fields are all documented at https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/Revisions/List?view=azure-devops-rest-5.0
-                case AzureDevOpsField.Title:           return "System.Title";
```

Comment:
The original version might be slightly more readable due to the column alignment, but this seems OK